io.github.openstreamorg.music.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.openstreamorg.music
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: openstreammusic
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/openstreammusic/flatpak-node/cache
+        npm_config_cache: /run/build/openstreammusic/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      # Change directory to 'desktop'
+      - rm -r index.html
+      - mv desktop/* .
```

Comment:
why do you need to do that?

*hint* there is `subdir`.

Reply:
How can I use subdir?